problem-24/README.md

```diff
@@ -0,0 +1,11 @@
+# Task 24: Disable button using data binding (lv 1)
+
+## Description
+Update the 'CounterViewComponent' by disabling the button using data binding. Bind the button to the boolean variable 'disableButton'.
```

Comment:
there is no CounterViewComponent in this project, and the project not even compile. 

Think though again

Reply:
Resolved - changed name and removed bug